dev/com.ibm.ws.cdi.internal/src/com/ibm/ws/cdi/internal/interfaces/CDIUtils.java

```diff
@@ -393,4 +395,19 @@ public static boolean isWeldProxy(Object obj) {
         boolean result = isWeldProxy(clazz);
         return result;
     }
+
+    /**
+     * Returns all interceptor bindings on the invocation context.
+     *
+     * @return a set of interceptor bindings from the invocation context.
+     * @throws IllegalArgumentException if InvocationContext is not an instance of org.jboss.weld.interceptor.proxy.AbstractInvocationContext;
+     */
+    public static Set<Annotation> getInterceptorBindingsFromInvocationContext(InvocationContext invocationContext) throws IllegalArgumentException {
```

Comment:
Need to explain what interceptor bindings are expected to be on the invocation context.